src/multiParamManager.js

```diff
@@ -5,6 +5,23 @@ var css = require('./universal-dapp-styles')
 var copyToClipboard = require('./app/ui/copy-to-clipboard')
 var remixLib = require('remix-lib')
 var txFormat = remixLib.execution.txFormat
+var $ = require('jquery')
+
+function doValidation (that, id) {
```

Comment:
we also need to apply this function just before sending the transaction

Reply:
do you know what event triggers the sending of the transaction so I can add the validation there?

Reply:
the sending happens here https://github.com/ethereum/remix-ide/blob/master/src/multiParamManager.js#L134

Reply:
in `doValidation` , should the other input be reset as well?
Note that there could be more than 1 param, for instances:
```
 function tes (uint _p, string _oo, address _o) {}
```

Reply:
the other input be reset? I'm not sure what you mean, and why would there be more than one param?

Reply:
`function sampleFn (uint _p, string _oo, address _o) {}` has 3 inputs
so the panel looks like
![screenshot from 2018-07-05 13 09 13](https://user-images.githubusercontent.com/6940742/42320201-d009e77a-8054-11e8-8983-ecd4fa7e03b3.png)
if the validation success, the red border should be removed on all the inputs